src/components/IntlSelectForm.js

```diff
@@ -1,16 +1,34 @@
+/*  I refactored this to be a sub component for the new SelectionForm component.
```

Comment:
I refactored this to be a sub component for the new `SelectionForm` component.
Once again I realized that to keep the code DRY I needed to refactor the selection components to make them more reusable.  This now gets passed into `SelectionForm` component to render the proper content for the select form elements.